pkg/db/db_test.go

```diff
@@ -0,0 +1,109 @@
+package db
+
+import (
+	"database/sql"
+	"fmt"
+	"os"
+	"testing"
+)
+
+var (
+	testDB    *Database
+	testDBErr error
+)
+
+func TestMain(m *testing.M) {
+	setup()
+	c := m.Run()
+	teardown()
+	os.Exit(c)
+}
+
+func setup() {
+	conn, err := sql.Open("sqlite3", ":memory:")
+	testDB = (*Database)(conn)
+	testDBErr = err
```

Comment:
This might be a nitpick, but I'll mention it either way. 😄 

I understand why you are not checking `err` here. However, if `err  != nil`, line 36 will fail and the test will exit with a message`failed to create table`, which is misleading.

IMO we should "fail fast" here.